app/jobs/rating_updating_job.rb

```diff
@@ -0,0 +1,8 @@
+class RatingUpdatingJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(book_id)
```

Comment:
activejob, they support global serialization object for you, so you can pass real object instead of id.